src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -367,10 +367,16 @@ def setupProcessingTask(self, procTask, taskType, inputDataset=None, inputStep=N
         procTaskCmsswHelper = procTaskCmssw.getTypeHelper()
         procTaskStageHelper = procTaskStageOut.getTypeHelper()
 
-        if 'Multicore' in taskConf and taskConf['Multicore'] > 0:  # used for StepChain overrides
-            procTaskCmsswHelper.setNumberOfCores(taskConf['Multicore'])
-        else:
-            procTaskCmsswHelper.setNumberOfCores(self.multicore)
+        # StepChain overrides
+        multicore = self.multicore
+        eventStreams = self.eventStreams
+        if 'Multicore' in taskConf and taskConf['Multicore'] > 0:
+            multicore = taskConf['Multicore']
+        if 'EventStreams' in taskConf and taskConf['EventStreams'] >= 0:
+            eventStreams = taskConf['EventStreams']
```

Comment:
Sorry never mind at the coments. I was looking at the wrong code. It seems you merged already Eric's change here.